modules/anti-ghostping/events/messageDelete.js

```diff
@@ -6,6 +6,8 @@ module.exports.run = async function (client, msg) {
     if (!messageWithMentions[msg.id]) return;
     const moduleStrings = client.configurations['anti-ghostping']['config'];
     if (messageWithMentions[msg.id].author.bot) return;
+    if (moduleStrings.ignoredChannels.includes(msg.channel.id)) return;
+    if (msg.member.roles.cache.find(r => moduleStrings.ignoredRoles.includes(r.id))) return;
```

Comment:
I don't think these checks are needed here, as `messageWithMentions` only includes this message if checks got passed (see: `messageCreate.js`, line 7-10)